### ISSUE TITLE
Style: Add divider before title

### ISSUE THREAD

**Issue by kilchenmann** [dasch-swiss/dsp-docs#6]
In the user guide we have only three topics with long markdown files and a lot of information. It's hard to follow or to see the different sections in each of these md files.

![Screenshot 2019-10-04 at 15 14 09](https://user-images.githubusercontent.com/4253580/66210617-89b29b00-e6ba-11e9-85f2-a76cbc8d79a4.png)

Can we add divider before a new title? Just add `---`

---

Or should we update some css and add divider/borders before a title (h2/h3)?

**Comment by flavens** [comment]
syntax to use: [horizontal rules](https://www.markdownguide.org/basic-syntax/#horizontal-rules)
